data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,462 @@
+			`	"What we need is a skilled pilot with a jump drive to visit some locations we cannot easily access."`
+				goto questions
+			label help precious opinions  
+			`	"We have some leads in human and Korath space we need you to follow. The Quarg at Alta Hai had me talk to scientists on <destination>. They say they can help, but they need a sample of the microtransceivers. I know the humans gave you a Jump Drive; I need you to use it to deliver the microtransceivers."`
+			`	She hands you a data card, "This has some info on it that you might find useful. Right now though the Quarg in Korath space might eliminate some leads, and add new ones. Follow the leads. Once you have proof the kidnappings are an organized effort, return here, and we will pay you 2,000,000 credits."`
+			choice
+				`	"I'll start right away."`
+					accept
+				`	"No, I'm not interested."`
+					decline
+
+# Must be after the Quarg and Korath intro missions:
+mission "ZZ Hai Rescue [A03] Microtransceiver Identification"
+	landing
+	name "Missing Researchers"
+	description "Fly to <destination> to talk to Tarazed Shipyards about the microtransceivers."
+	source "Kuwaru Efreti"
+	destination "Wayfarer"
+	to offer
+		has "Hai Rescue [A02] Microtransceiver Identification: done"
+	on offer
+		log `The Korath identified the microtransceivers as using the same materials and design principles as their own technology, but extremely primitive and poorly-made. The Quarg provided a data card that Tarazed Shipyards may be able to use.`
+		conversation
+			`After you land, a hovering gray platform lowers to the entrance of your ship. You step on, and it raises you to a higher level of the ringworld where many Quarg and a few Korath are present. You find the first Quarg who will talk to you and show it the vial of microtransceivers.`
+			`	The Quarg makes a series of wheezing sounds which approximate a laugh. "I am merely a pilot. I know nothing of microtransceivers. That is the technology of the Korath, and I will take you to one who is knowledgeable."`
+			`	You pass through another two grey platform transports and reach a warm cave-like structure with a group of Korath and Quarg. From the holograms and equipment, it appears they're discussing science. One hologram looks like a blood cells attached to a tiny machine; this may be a microscale medicine laboratory.`
+			`	The Quarg and Korath discuss your microtransceivers for a few minutes, placing them in various instruments.`
+			`	Finally you have your answer. A Quarg scientist tells you, "These appear to be Korath in origin, but makeshift and primitive. Even the Korath Exiles are more advanced. Those who made these have little resources and simple machinery. This will tell you how to detect these microtransceivers," it hands you a standard data card compatible with human computers.`
+			`	"Take this to Tarazed Shipyards."`
```

Comment:
I always figured that the Exiles, being a smaller group that was more attached to their technology, likely had a higher proportion of scientists and the like, and put the effort in to make sure they absconded with everything they could. Whether that's data, schematics, prototypes, etc.
Whereas the Efreti are really in an ongoing humanitarian crisis, and practically speaking, even if they have those things, they've given up some stuff, and a lot of other things are likely buried under the weight of administration.

Reply:
That's reasonable. Here's the suggestion with my other edits but the sentence added back in:
```suggestion
			`	Finally, you have your answer. A Quarg scientist tells you, "These appear to be Korath in origin, but makeshift and primitive. Even the Korath Exiles are more advanced. Those who made these have little resources and simple machinery. This will tell you how to detect these microtransceivers," it explains, handing you a standard data card compatible with human computers. "Take this to Tarazed Shipyards."`
```